Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -9504,6 +9504,49 @@
 			"name": "Gunslinger",
 			"mental": true,
 			"base_points": 25,
+<<<<<<< HEAD
```

Comment:
This isn't valid... part of the merge conflict text

Reply:
I must have missed that part. It should be fixed now.